src/components/Table/TableStyles.tsx

```diff
@@ -42,6 +42,8 @@ export interface GridStyle {
   firstRowMessageCss?: Properties;
   /** Applied on hover if a row has a rowLink/onClick set. */
   rowHoverColor?: Palette | "none";
+  /** Applied on hover to a cell TextFieldBase */
+  rowEditableCellBorderColor?: Palette;
```

Comment:
@0ces afaict we're not really setting this on "the cell", but instead pushing it down into the child `TextField`'s DOM? If so, maybe it shouldn't really be a "table style" and instead each component can know how to style itself has "editable" or "borderless: unless-hovered" or what not.